lib/motion/configuration.rb

```diff
@@ -59,16 +59,14 @@ def initialize
       Rails.application.key_generator.generate_key("motion:secret")
     end
 
+    option :revision_paths do
+      require "rails"
+
+      paths
```

Comment:
This is a bit of a nitpick, but I would rather just inline `paths` here than have another method. Users will still be able to access it's value by calling the `revision_paths` method.